keps/sig-apps/2214-indexed-job/README.md

```diff
@@ -424,64 +424,60 @@ _This section must be completed when targeting beta graduation to a release._
 
 * **What specific metrics should inform a rollback?**
 
+  - job_sync_duration_seconds shows significantly more latency for Indexed Jobs.
+  - job_sync_total shows errors for Indexed Jobs only.
+  - job_finished_total shows that Indexed Jobs don't finish.
+
 * **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
-  Describe manual testing that was done and the outcomes.
-  Longer term, we may want to require automated upgrade/rollback tests, but we
-  are missing a bunch of machinery and tooling and can't do that now.
+
+  Manual test:
+  
+  1. Deploy k8s 1.21 cluster
+  1. Upgrade to 1.22
+  1. Create Indexed Job with big number of completions and pods that run for ~10min.
+  1. Downgrade to 1.21. Verify that no new pods are created for the Indexed Job.
+  1. Upgrade to 1.22. Verify that new pods are created for Indexed Job.
 
 * **Is the rollout accompanied by any deprecations and/or removals of features, APIs, 
 fields of API types, flags, etc.?**
-  Even if applying deprecation policies, they may still surprise some users.
+
+  No
 
 ### Monitoring Requirements
 
 _This section must be completed when targeting beta graduation to a release._
 
 * **How can an operator determine if the feature is in use by workloads?**
-  Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
-  checking if there are objects with field X set) may be a last resort. Avoid
-  logs or events for this purpose.
+
+  - job_sync_total has values for the label completion_mode=Indexed.
 
 * **What are the SLIs (Service Level Indicators) an operator can use to determine 
 the health of the service?**
-  - [ ] Metrics
-    - Metric name:
-    - [Optional] Aggregation method:
-    - Components exposing the metric:
-  - [ ] Other (treat as last resort)
-    - Details:
+
+  - [x] Metrics
+    - Metric name (all new): job_sync_duration_seconds, job_sync_total, job_finished_total
+    - Components exposing the metric: kube-controller-manager
 
 * **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
-  At a high level, this usually will be in the form of "high percentile of SLI
-  per day <= X". It's impossible to provide comprehensive guidance, but at the very
-  high level (needs more precise definitions) those may be things like:
-  - per-day percentage of API calls finishing with 5XX errors <= 1%
-  - 99% percentile over day of absolute value from (job creation time minus expected
-    job creation time) for cron job <= 10%
-  - 99,9% of /health requests per day finish with 200 code
+
+  - per-day percentage of job_sync_total with label result=error <= 1%
+  - 99% percentile over day for job_sync_duration_seconds is:
+    - <= 5s for single pod Jobs 
+    - <= 1m for parallel jobs with a few tens of pods.
+    - <= 1h for parallel jobs with a few thousands of pods.
```

Comment:
That 1h is definitely too long for a single sync.

Reply:
not if you consider https://github.com/kubernetes/community/blob/master/sig-scalability/slos/slos.md#steady-state-slisslos

Reply:
I agree that 1h is waay to much (and I don't really see the relevance with the link you pasted Aldo).

Also see - my comment above (it would help addressing this one too).